doc/classes/RichTextLabel.xml

```diff
@@ -24,10 +24,17 @@
 			<param index="3" name="color" type="Color" default="Color(1, 1, 1, 1)" />
 			<param index="4" name="inline_align" type="int" enum="InlineAlignment" default="5" />
 			<param index="5" name="region" type="Rect2" default="Rect2(0, 0, 0, 0)" />
+			<param index="6" name="key" type="Variant" default="null" />
+			<param index="7" name="pad" type="bool" default="false" />
+			<param index="8" name="tooltip" type="String" default="&quot;&quot;" />
+			<param index="9" name="size_in_percent" type="bool" default="false" />
 			<description>
 				Adds an image's opening and closing tags to the tag stack, optionally providing a [param width] and [param height] to resize the image, a [param color] to tint the image and a [param region] to only use parts of the image.
 				If [param width] or [param height] is set to 0, the image size will be adjusted in order to keep the original aspect ratio.
 				If [param width] and [param height] are not set, but [param region] is, the region's rect will be used.
+				If [param key] is set, image can be modified with [method update_image].
+				If [param pad] is set, and image is smaller than the size specified by [param width] and [param height], instead of upscaling the image padding is added to match the size.
```

Comment:
```suggestion
				If [param pad] is set, and the image is smaller than the size specified by [param width] and [param height], the image padding is added to match the size instead of upscaling.
```